scripts/build-rir.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -xe
+
+SRC_DIR=${SRC_DIR:-$PWD}
+BUILD_DIR=${BUILD_DIR:-$PWD/rkt-build}
+RKT_BUILDER_ACI=${RKT_BUILDER_ACI:-coreos.com/rkt/builder:v0.10.0}
+
+mkdir -p $BUILD_DIR
+
+rkt run \
+    --volume src-dir,kind=host,source=$SRC_DIR \
+    --volume build-dir,kind=host,source=$BUILD_DIR \
+    --interactive \
+    --insecure-skip-verify \
+    $RKT_BUILDER_ACI \
```

Comment:
I wasn't sure how to handle this since currently the ACI isn't published anywhere (nor am I sure on how that will end up working). Since this PR expects it to be a local ACI to be imported into the CAS can I leave this for now?